dashboard/app/controllers/api/v1/schools_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::SchoolsController < ApplicationController
+  # GET /api/v1/school/<school_district_id>/<school_type>
+  def index
+    schools = School.where(school_district_id: params[:school_district_id], school_type: params[:school_type])
```

Comment:
Any reason we shouldn't be sanitizing `params[:school_district_id]` and `params[:school_type]`?
http://apidock.com/rails/ActiveRecord/QueryMethods/where